pkg/secrethub/secret.go

```diff
@@ -47,6 +47,7 @@ type SecretService interface {
 	//  }
 	EventIterator(path string, _ *AuditEventIteratorParams) AuditEventIterator
 	// ListEvents retrieves all audit events for a given secret.
+	// Deprecated: Use iterator function instead.
```

Comment:
This one we cannot deprecate yet, as the audit subject type filter is not supported yet on the iterator.

Reply:
Fixed!